components/Navbar/CreateContestForm.tsx

```diff
@@ -54,12 +79,63 @@ export default function CreateContestForm({
     cover_picture: null,
     prize: '',
     datesOption: '',
-    uploadPhaseDate: uploadDate,
-    votingPhaseDate: votingDate,
+    upload_phase_end: uploadDate,
+    voting_phase_end: votingDate,
+    automated_dates: automatedDates,
   }
 
-  const handleSubmit = (values: typeof initialValues) => {
-    console.log(values)
+  const handleSubmit = async (contestValues) => {
+    const {
+      title,
+      description,
+      cover_picture,
+      prize,
+      upload_phase_start,
+      upload_phase_end,
+      voting_phase_end,
+      automated_dates,
+    } = contestValues
+
+    try {
+      const picture = {
+        user: 'test@test.com',
+        picture_path: cover_picture,
+      }
+
+      const createdPicture = await createPicture({
+        variables: { picture },
+      })
+
+      contestValues.cover_picture = {
+        id: createdPicture.data.create_picture.id,
+      }
```

Comment:
is this doing anything? `contestValues` isn't used after this

Reply:
removed